packages/osd-opensearch/src/artifact.js

```diff
@@ -40,8 +39,11 @@ const { createHash } = require('crypto');
 const path = require('path');
 
 const asyncPipeline = promisify(pipeline);
-const DAILY_SNAPSHOTS_BASE_URL = '';
+const DAILY_SNAPSHOTS_BASE_URL = 'https://artifacts.opensearch.org/snapshots/core/opensearch';
+// TODO: [RENAMEME] currently do not have an existing replacement
```

Comment:
Do we have an Issue open to address these TODOs? I think our philosophy should be to add as much detail to TODOs as we can so that context isn't lost over time (or do our best to never add TODOs). And if possible, open an Issue for it as well.